sourced/ml/tests/test_uast_to_role_id_pairs.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+from bblfsh import BblfshClient
+
+from sourced.ml.algorithms import Uast2RoleIdPairs, NoopTokenParser
+from sourced.ml.tests.models import SOURCE_PY
+
+
+class Uast2NodesBagTest(unittest.TestCase):
+    def setUp(self):
+        self.uast2role_id_pairs = Uast2RoleIdPairs(token_parser=NoopTokenParser())
+        self.uast = BblfshClient("0.0.0.0:9432").parse(SOURCE_PY).uast
+
+    def test_result(self):
+        correct = [('ModuleSpec', 'BODY IF THEN'),
```

Comment:
Are you sure the next crazy Babelfish version will not add `SPACE SEPARATED` roles? :)

Reply:
Yes. It this case I will fix it easily.

Reply:
We don't want to fix, we want to write once and live happily solving next problems :) How about changing to " | " (two spaces and |)

Reply:
ok